web/src/components/navbar/NavBar.test.tsx

```diff
@@ -102,6 +102,7 @@ describe("<NavBar />", () => {
   beforeEach(() => {
     jest.resetAllMocks();
     useMockRouter({});
+    useMockUserData({});
   });
```

Comment:
@somabadri The `NavBar` component uses dependency injection/ passes business as a Prop, still not clear as to why useMockUserData({}) is needed. Shouldn't we be using the generator fn to create the business and pass it in as an argument?

Reply:
I think Soma answers this in the "resolved" comment.